src/main/java/com/exactpro/th2/conn/MicroserviceMain.java

```diff
@@ -302,7 +298,7 @@ private static void createPackAndPublishPipeline(Direction direction, Flowable<C
                 .subscribe(batch -> {
```

Comment:
We don't need to have `refCount` and `publish` above. We can simple call subscribe on the flowable